src/css/components/_text-track.scss

```diff
@@ -24,3 +24,7 @@
 .vjs-subtitles { color: #fff /* Subtitles are white */; }
 .vjs-captions { color: #fc6 /* Captions are yellow */; }
 .vjs-tt-cue { display: block; }
+
+video::-webkit-media-text-track-display {
+  @include transform(translateY(-45px));
```

Comment:
Will do. I'll retest `em`, but expect to have the captions bleed into the video depending on the `font-size` set by the browser width.

Also, should the captions work like native `-webkit`, where they adjust for the control bar, or should they stay at a fixed (relative) position.

> Also, we should make this align with where we stick the emulated captions.

Don't follow.